hooks/accordion.php

```diff
@@ -5,7 +5,7 @@
  * behind our fixed header. See: https://wordpress.org/plugins/accordion-shortcodes/other_notes/
  */
 add_filter('shortcode_atts_accordion', function($atts) {
-    if (isset($atts['scroll']) || $atts['scroll']) {
+    if (isset($atts['scroll']) = $atts['scroll']) {
```

Comment:
Ref: accordion shortcode documentation noting how to use `false` or `true` or a value for the scroll:

https://wordpress.org/plugins/accordion-shortcodes/other_notes/